Sources/SecureXPC/Server/XPCMachServer.swift

```diff
@@ -12,7 +12,11 @@ import Foundation
 /// In the case of this framework, the XPC Service is expected to be communicated with by an `XPCMachClient`.
 internal class XPCMachServer: XPCServer, NonBlockingStartable {
     
+    /// Name of the service.
     private let machServiceName: String
+    /// Receives new incoming connections, created once the server is started.
+    private var machService: xpc_connection_t?
```

Comment:
I think we should used the term-of-art for this, which IIRC is "listener connection". From the `xpc_connection_create(3)` man page:

> If the service is a MachService advertised by launchd(8), then the listener connection for the named service will receive the new connection in its event handler.

[`xpc_connection_create_mach_service(_:_:_:)`](https://developer.apple.com/documentation/xpc/1448783-xpc_connection_create_mach_servi)

> If the XPC_CONNECTION_MACH_SERVICE_LISTENER flag is given to this method, then the connection returned will be a listener connection.****